src/utils/formatValue.ts

```diff
@@ -9,16 +10,31 @@ export type ValueFormatter = (value: number, unit?: string, options?: FormatOpti
 export const unitLookupKey = unit => {
```

Comment:
Not sure why we had unit here, but it should be units -- we can update this later